manifests/m/MacroDeck/MacroDeck/2.12.0/MacroDeck.MacroDeck.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-22572-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MacroDeck.MacroDeck
+PackageVersion: 2.12.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install
+InstallerSwitches:
+  Silent: /VERYSILENT /SUPPRESSMSGBOXES /CLOSEAPPLICATIONS
+  SilentWithProgress: /SILENT /SUPPRESSMSGBOXES /CLOSEAPPLICATIONS
```

```suggestion
InstallerSwitches:
  Custom: /SUPPRESSMSGBOXES
```
`/SILENT` and `/VERYSILENT` are automatically added when inno is specified. `/CLOSEAPPLICATIONS` is generally not advised since it could lead to user confusion. It's better to add ExpectedReturnCodes for when the application is in use